calculator.rb

```diff
@@ -38,10 +48,12 @@ def divide(operand_1, operand_2)
   result = multiply(operand_1, operand_2)
 when "/"
   result = divide(operand_1, operand_2)
-when "**"
-  result = operand_1 ** operand_2
+when "^"
+  result = exponent(operand_1, operand_2)
+when "sqrt"
+  result = root(operand_1)
 else
   puts "I have no idea what you intended to do here."
 end
 
-puts result
\ No newline at end of file
+print result
```

Comment:
Nitpick: I think this will leave the terminal tacked on to the end of the output, won't it?

Reply:
Actually, it doesn't.